deploy_scripts/substitute_env.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+for TEMPLATE in pictor/config.py.deploy \
```

Comment:
We have the application name as an environment variable in Travis, right? So we could do 
```suggestion
for TEMPLATE in &{APPLICATION_NAME}/config.py.deploy \
```